src/transaction/TransactionController.ts

```diff
@@ -713,21 +723,28 @@ export class TransactionController extends BaseController<
       throw new Error('No sign method defined.');
     }
 
-    const gasPrice = getIncreasedPriceFromExisting(
-      transactionMeta.transaction.gasPrice,
-      CANCEL_RATE,
-    );
+    const gasPrice =
+      gasValues?.gasPrice ||
```

Comment:
We're not validating that the new custom price is at least 10% higher than the previous one, so this might fail on the network. That doesn't seem ideal. Do you think it would make sense to call `getIncreasedPriceFromExisting` even if a custom price is provided, and use that as a minimum?

Reply:
I see what you're saying... I guess my thinking was that the client would ensure the newly suggested price is at least 10% higher. Are you suggesting we utilise `getIncreasedPriceFromExisting` to get the minimum and then compare that to the suggested price and throw if the newly suggested price is less than that?